src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -3915,4 +3915,8 @@ In certain complex graphs or host program scenarios, Automatic mode may cause in
   <data name="PublishPackageRetainFolderStructureToggleButtonText" xml:space="preserve">
     <value>Retain folder structure</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="PreferencesViewEnablePanelNodes" xml:space="preserve">
+    <value>Enable Panel nodes</value>
```

Comment:
I'm not happy with the protogeometry class name - `Panel` as it actually represents a panel-mesh. I'll probably change it after I get some initial feedback. On a similar note, I think `Enable Paneling nodes` would be more appropriate here.